src/crawler/crawlers/arso_water_slovenija/crawler.js

```diff
@@ -1,159 +1,173 @@
+     * @param {Array} data 
+     * @param {Date} from_time 
+     */
     findData($, data, from_time) {
-        let get_out = false
-        let self = this
-        $('body>table>tbody>tr>td.vsebina>table.podatki>tbody>tr').each(function() { // get data
-            let new_data = []
-            let iter_num = 0
+        let get_out = false;
+        let self = this;
+        $('body>table>tbody>tr>td.vsebina>table.podatki>tbody>tr').each(function () { // get data
+            let new_data = [];
+            let iter_num = 0;
     
             $(this).find('td').each(function () {
-                var item = $(this).text()
+                let item = $(this).text();
     
                 if (iter_num == 0) {
-                    item = String(moment(item, self.time_parse).format())
+                    item = String(moment(item, self.time_parse).format());
                     if (from_time >= Date.parse(item)) {
-                        get_out = true
-                        return false
-                    }
-                }
-                new_data.push(item)
-                iter_num += 1
+                        get_out = true;
+                        return false;
+                    };
+                };
+                new_data.push(item);
+                iter_num += 1;
             });
     
             if (get_out) {
-                return false
+                return false;
             }
-            if(self.water_type == 'gw' && new_data[1] == '-' && new_data[2] == '-') {
+
+            //console.log(new_data)
+            if (self.water_type == 'gw' && new_data[1] == '-' && new_data[2] == '-') {
                 // do not write when no data for ground water
             } else if (self.water_type == 'sw' && new_data[1] == '-' && new_data[2] == '-' && new_data[3] == '-') {
                 // do not write when no data for surface water
             } else {
-                data.push(new_data)
-            }
+                data.push(new_data);
+            };
             
-        })
-        return data
+        });
+        return data;
     }
 
-    // check last date of data
+    /**
+     * Check last date.
+     * @param {String} dat 
+     */
     checkLastDate(dat) {
         let lines = dat.trim().split('\n');
         let lastLine = lines.slice(-1)[0];
     
         let splitLine = lastLine.split(',');
         let from_time = Date.parse(splitLine.slice(0)[0]);
-        
-        return from_time
+
+        return from_time;
     }
 }
+module.exports = ArsoWaterSloveniaCrawler;
 
-var cr_sur = new ArsoWaterSloveniaCrawler(water_type='sw', URL_file='linksSurface.txt', data_file='./data_surfacewater/')
-var cr_gr = new ArsoWaterSloveniaCrawler(water_type='gw', URL_file='linksGround.txt', data_file='./data_groundwater/')
-
-//cr_sur.getURLs()
-//cr_gr.getURLs()
+let cr_sur = new ArsoWaterSloveniaCrawler(water_type='sw', url_file='linksSurface.txt', data_file='./data_surfacewater/');
```

Comment:
Implement this in main file; do not instantiate in this file. This is only a library.